data/formats-data.js

```diff
@@ -3787,7 +3794,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	cranidos: {
-		randomBattleMoves: ["headsmash", "rockslide", "earthquake", "zenheadbutt", "firepunch", "rockpolish", "crunch"],
+		randomBattleMoves: ["headsmash", "rockslide", "earthquake", "superpower", "zenheadbutt", "thunderpunch", "rockpolish", "pursuit"],
```

Comment:
fire punch should be kept - it covers stuff like ferroseed very well, plus it's boosted by sheer force.

Reply:
oh yes sheer force boosts it. it can stay but holy crap thats a lot of coverage option